test/compiletimes.jl

```diff
@@ -12,8 +17,27 @@ end
 
 path = joinpath(pkgdir(Pluto), "sample", "Basic.jl")
 
-@timeit TOUT "SessionActions.open" nb = Pluto.SessionActions.open(🍭, path; run_async=true)
+@timeit TOUT "SessionActions.open" nb = @eval Pluto.SessionActions.open(🍭, path; run_async=true)
 
 wait_for_ready(nb)
 
-@timeit TOUT "SessionActions.shutdown" Pluto.SessionActions.shutdown(🍭, nb; async=true)
+@timeit TOUT "SessionActions.shutdown" @eval Pluto.SessionActions.shutdown(🍭, nb; async=true)
+
+# According to SnoopCompile, this is a big part of the time for `Pluto.run()`.
+# However, it's very tricky to measure this via the `Pluto.run` below.
+@timeit TOUT "Configuration.from_flat_kwargs" @eval Pluto.Configuration.from_flat_kwargs()
+
+HTTP.get("http://github.com")
```

Comment:
This is in order not to count the compilation time of `HTTP.get` in the following block, right?
A comment stating this would be helpful.